vignettes/miaSim.Rmd

```diff
@@ -42,5 +44,6 @@ library("miaSim")
 # Models for simulating microbiome data sets 
 
 The generalized Lotka-Volterra model fitted to time-series estimates
-microbial population dynamics and relative rates of interaction.The abundance matrix 
-stored in `assays` of SummarizedExperiment object is main task to find with this model.
+microbial population dynamics and relative rates of interaction.The abundance 
```

Comment:
remember to add space after period always - it is a standard writing convention..